recipes/cuda-cudart/meta.yaml

```diff
@@ -0,0 +1,290 @@
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libcudart.so.{{ version.split(".")[0] }}  # [linux]
+        - if not exist %LIBRARY_BIN%\cudart64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA Runtime native Libraries
+      description: |
+        CUDA Runtime native Libraries
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-cudart-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("cuda-cudart", max_pin="x") }}
+    files:
+      - lib/libcu*.so  # [linux]
+      - lib/pkgconfig  # [linux]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("cuda-cudart", exact=True) }}
+        - {{ pin_subpackage("cuda-cudart-dev_" + target_platform, exact=True) }}
+        - {{ pin_subpackage("cuda-cudart-static", exact=True) }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}                    # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcudart.so           # [linux]
+        - test -f $PREFIX/lib/pkgconfig/cuda-*.pc    # [linux]
+        - test -f $PREFIX/lib/pkgconfig/cudart-*.pc  # [linux]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA Runtime native Libraries
+      description: |
+        CUDA Runtime native Libraries
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-cudart-dev_{{ target_platform }}
+    build:
+      noarch: generic
+      run_exports:
+        - {{ pin_subpackage("cuda-cudart", max_pin="x") }}
+    files:
+      - targets/{{ target_name }}/include/*                     # [linux]
+      - targets/{{ target_name }}/include/cooperative_groups/*  # [linux]
+      - targets/{{ target_name }}/lib/*.so                      # [linux]
+      - Library\include                                         # [win]
+      - Library\lib\cuda.lib                                    # [win]
+      - Library\lib\cudadevrt.lib                               # [win]
+      - Library\lib\cudart.lib                                  # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - cuda-cccl_{{ target_platform }}
```

Comment:
Given we're just copying around headers / libraries for this package, why is this required in the build section?

Also, if this is needed it should be versioned.